traits/trait_types.py

```diff
@@ -20,10 +20,15 @@
 import operator
 import re
 import sys
+try:
+    from os import fspath
+except ImportError:
+    fspath = None
 from os.path import isfile, isdir
 from types import FunctionType, MethodType, ModuleType
 import uuid
 
+
```

Comment:
Unrelated spacing change: please revert. (Our usual conventions are for only a single line between import blocks.)